### PR TITLE
phenotype import needed updates -e99

### PR DESCRIPTION
**Pull request by ima23**
- the first 3 commits are already merged part of PR https://github.com/Ensembl/ensembl-variation/pull/464
- fix for CGC json updates
- fix path for orphanet

### CODE REVIEW

File: modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/BasePhenotypeAnnotation.pm [hunk 1]
```diff
@@ -233,6 +229,24 @@ sub debug {
 }
 
 
+=head2 skip_sets
+
+Arg [1]    : boolean $skip_sets (optional)
+             The new skip_sets flag
+Example    : $kip_sets = $obj->skip_sets()
```
Comment (at7) [inline modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/BasePhenotypeAnnotation.pm]: typo $kip_sets -> $skip_sets

**Comment by at7** [comment]
There are conflicts with BasePhenotypeAnnotation.pm. I can merge the PR after the conflicts have been resolved.

**Comment by ima23** [comment]
@at7, the conflict is resolved. Please feel free to merge. Thanks.

**Comment by at7** [comment]
merged into master and release/99

**Pull request merged**